frontend/packages/zoo-api/src/queries.ts

```diff
@@ -394,54 +452,24 @@ export interface ActiveIncident extends Node {
   color: string;
 }
 
-export interface Resource extends Node {
+export interface Dependancy extends Node {
```

Comment:
Could you replace Depend**a**ncy by Depend**e**ncy in this MR?

Reply:
Sorry, I am little bit dyslexic;) Fixed